docs/zh/standards/uigf.md

```diff
@@ -4,188 +4,334 @@ order: 1
-| [`v2.4`](uigf-legacy-v2.4.md) | 新增 `info.region_time_zone` 支持时区处理       | v2.4 and lower |
-| `v3.0`                        | 新增 集录祈愿类型支持       | v3.0 and lower |
-
-### v3.0 更新内容
-
-* `gacha_type` 增加新枚举项
-  * 在 `gacha_type` 枚举新增值为 `500` 的项，用于表示集录祈愿类型
-
-## `info` 字段说明
-
-### `region_time_zone`
-
-由于在获取祈愿记录时得到的`time`为服务器时间，为了准确判断时间的时区偏移，引入此字段。
-
-与 SRGF 不同，由于无法直接从服务器获取`region_time_zone`，在导出方未提供此字段时，需要根据 `uid` 进行推断。
-
-#### 映射关系
-
-| `uid`首个字符 | `region_time_zone` | 游戏服务器                             |
-|-----------|--------------------|-----------------------------------|
-| `'6'`     | `-5`               | os_usa                            |
-| `'7'`     | `1`                | os_euro                           |
-| 剩余情况      | `8`                | os_cht, os_asia, cn_gf01, cn_qd01 |
-
-App 不应假定 `region_time_zone` 的值为上表中给出的值，应具有处理非标准 `region_time_zone` 值的能力。
-若 `region_time_zone` 的值与 `uid` 推断结果不一致，则优先选择 `region_time_zone` 给出的值。
-
-## `list` 字段说明
+| 版本   | 说明                            | 兼容             |
+|--------|--------------------------------|----------------|
+| Lower  | 低版本的更新日志请查看历史版本    | v3.0 and lower |
+| `v4.0` | 合并 SRGF，新增绝区零抽卡格式支持 | v4.0           |
 
-### `id`
+## 前言
 
-物品内包含了一项较为特殊的字段: `id`，为原神官方 API 中包含的，代表每条抽卡记录唯一性的 `id`。App 导出 UIGF 时
-* 需要确保每个物品都有一个有效的唯一 `id` 
-* 若有记录中不包含`id`，则应从下一个自带有效 `id` 的物品开始，为每条缺失`id`字段的数据补全`id`。
-赋值数据向前（时间排序）依次递减，每次递减的值应保持为 `1`
-
-### `gacha_type`
-
-由于存在会共享保底与概率的卡池，所以需要一个额外的字段来界定  
-我们在 `UIGF` 的所有格式中注入了 `uigf_gacha_type` 字段  
-在导出到 `UIGF` 格式时需要注意添加对应的 `uigf_gacha_type` 字段。
-
-#### 映射关系
-
-| `uigf_gacha_type` | `gacha_type`   |
-|-------------------|----------------|
-| `100`             | `100`          |
-| `200`             | `200`          |
-| `301`             | `301` or `400` |
-| `302`             | `302`          |
-| `500`             | `500`          |
-
-### `item_id`
-
-物品游戏内ID，你可以通过 [UIGF API](../api.md) 获取这一数据。
+为了统一不同 App, 不同游戏，不同账号间的抽卡记录导入导出行为，我们决定将所有支持的游戏抽卡格式合入 UIGF 中。
+不同的游戏，不同的账号将能够以单个文件或字符串的形式进行表示，导入与导出操作对用户而言将变的史无前例的简单。
 
 ## Json Schema
 
 > UIGF-Org 提供下述 Json Schema 以用于验证资料结构的正确性。
 
 ::: warning 注意字段类型
-请各位开发者务必尊重 Schema 内定义的字段类型。使用错误的类型可能会导致其它由强类型编程语言制成的工具在解析 Json 文件时产生错误，进而导致数据转移失败。
+开发者务必尊重 Schema 内定义的字段类型。使用错误的类型可能会导致其它由强类型编程语言制成的工具在解析 Json 文件时产生错误，进而导致数据转移失败。
 
-为了避免这类问题，我们建议您针对 UIGF 格式设计专用的 struct，或善用 `JsonNumberHandling.WriteAsString` 等方法。同时，设计相关的单元测试以确保导入导出的一致性。
+为了避免这类问题，我们建议您针对 UIGF 格式设计专用的数据结构。同时，设计相关的单元测试以确保导入导出的一致性。
 
 我们也提供 [UIGF 格式校验工具](https://schema.uigf.org/?schema=uigf)来帮助你校验 Json 文件。
 :::
 
 ```json
 {
-  "type": "object",
-  "properties": {
-    "info": {
-      "type": "object",
-      "properties": {
-        "uid": {
-          "type": "string",
-          "title": "导出记录的 UID"
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "properties": {
+        "info": {
+            "type": "object",
+            "properties": {
+                "export_timestamp": {
+                    "type": "string|integer",
```

Comment:
```suggestion
                    "type": "string",
```
I suggest using String in this field due to concerns I have said above.

Also, the timestamp in Swift is Double.
https://developer.apple.com/documentation/uikit/uievent/1613790-timestamp